omod/src/main/webapp/resources/scripts/navigator/navigatorHandlers.js

```diff
@@ -307,18 +314,20 @@ var clickedFieldHandler = function(fields, field, event) {
     var clickedFieldIndex = _.indexOf(fields, field);
     var shouldSelectClickedField = true;
     if(clickedFieldIndex > currentFieldIndex) {
-        for(var i=currentFieldIndex; i<clickedFieldIndex; i++) {
+        for(var i=currentFieldIndex == -1 ? 0 : currentFieldIndex; i<clickedFieldIndex; i++) {
```

Comment:
All the changes in this method were fixing an existing NPE that occurs if you click on "No" from the confirmation section.

Reply:
```suggestion
        var startIndex = currentFieldIndex == -1 ? 0 : currentFieldIndex;
        for(var i=startIndex; i < clickedFieldIndex; i++) {
```